### PR TITLE
Configure properly powerauth-webflow-client for JBoss / Wildfly

### PR DESCRIPTION
**Pull request by saalistaja**

### CODE REVIEW

File: docs/Deploying-Wildfly.md [hunk 1]
```diff
@@ -44,6 +46,7 @@ Similarly, Next Step contains the following configuration in `jboss-deployment-s
 ```
 
 The deployment descriptor requires configuration of the `com.wultra.powerauth.webflow.conf` and `com.wultra.powerauth.nextstep.conf` modules.
+Optionally the `com.wultra.powerauth.webflow-client.conf` module has to be configured also.
```
Comment (romanstrobl) [inline docs/Deploying-Wildfly.md]: Optionally and "has to" contradict themselves.

File: powerauth-webflow-client/src/main/webapp/WEB-INF/jboss-deployment-structure.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<jboss-deployment-structure xmlns="urn:jboss:deployment-structure:1.2">
+	<deployment>
+		<exclude-subsystems>
+			<!-- disable the resource-adapters subsystem to prevent the application's HSQLDB driver
+			from being included in application server JDBC drivers -->
+            <subsystem name="resource-adapters" />
```
Comment (romanstrobl) [inline powerauth-webflow-client/src/main/webapp/WEB-INF/jboss-deployment-structure.xml]: The formatting seems a bit off.

**Comment by romanstrobl** [comment]
Looks good now.

File: powerauth-webflow-client/src/main/webapp/WEB-INF/jboss-deployment-structure.xml (hunk 2 above)
Comment (saalistaja) [reply powerauth-webflow-client/src/main/webapp/WEB-INF/jboss-deployment-structure.xml]: Fixed.

The reason: I am using space chars only in files. For XML I've reconfigured IDEA to use tab char.

File: docs/Deploying-Wildfly.md (hunk 1 above)
Comment (saalistaja) [reply docs/Deploying-Wildfly.md]: Aaah, could be a little bit weird :). Changed.

**Pull request merged**